src/servers/graphql-old-server.ts

```diff
@@ -239,11 +232,11 @@ const resolvers = {
     noauthAddInvoice: async (_, { username, value }) => {
       const lnInvoice =
         value && value > 0
-          ? await addInvoiceForRecipient({
+          ? await Wallets.addInvoiceForRecipient({
```

Comment:
Does switching these out mean we can delete the old methods? Also are we sure the return types line up?

Reply:
These are the new methods, I just made the imports from wallets standard (using * rather than named imports)